src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java

```diff
@@ -74,12 +73,15 @@
     startDatePicker = createDatePicker();
     endDatePicker = createDatePicker();
     $$$setupUI$$$();
+    startDatePicker.getSettings().setDateRangeLimits(null, LocalDate.now());
+    startDatePicker.setDateToToday();
```

Comment:
I think we shouldn't set the default start date to any value...

Reply:
ok but it has to be before the current date right?

Reply:
That makes sense but I wonder if we could have problems with forms pulled from different time zones... Maybe a form is from a timezone with a positive offset that effectively marks it as being created "in the future" respect to the host's timezone.

We can sit on this one and wait for feedback from @lognaturel or @yanokwa to confirm this bit.

Reply:
Okay